### DISCUSSION TITLE
Windows 10 Search Bar Pane (not the search menu)

### DISCUSSION THREAD

**Discussion by ItzExvirnity** [valinet/ExplorerPatcher#2447] [Ideas]
I would like an option that you would just toggle on and then you would get a Windows 10 Search Bar pane on the taskbar and then toggle it off and _whoosh_, it's gone! If you click it, it will open the 11 search menu but without the search bar in the menu, when you type in it will not type on the Windows 11 Search Menu, the text will be on the search bar pane.

Something that would look like this          (made in paint.net)

![image](https://github.com/valinet/ExplorerPatcher/assets/118678848/849ae345-8905-4c65-ab70-50ffd79acc13)

**Comment by pyrates999** [comment]
I think that is beyond what EP was designed for.

**Reply by ItzExvirnity** [reply]
Makes a lot of sense, I am also thinking this, now being able to modify the search menu. Nah that won't work

**Reply by EdgeDev01** [reply]
that would work hey @pyrates999 forward this idea to valinet

**Reply by EdgeDev01** [reply]
or if you want you @Mattem300 you can fork this and add that feature along with the windows 10 Action Center flyout, and the windows 10 Search bar that shows up when the search icon is moved from icon to search box on windows 11

**Comment by milestprower92** [comment]
No longer possible in 22H2 and 23H2 because Microsoft has removed the code of the old search box from the old taskbar and there are no plans to have it added back due to the fact that the search icon does the same thing as the search box.

**Reply by zett1201** [reply]
it could also be for 21h2 only or maybe a 1:1 replica is made?? (its not out ik)

**Reply by pyrates999** [reply]
Try EverythingToolBar: https://github.com/srwi/EverythingToolbar

**Reply by milestprower92** [reply]
only 22000's explorer has the searchbox
